lib/index.js

```diff
@@ -1,4 +1,5 @@
-module.exports = require('./api');
+module.exports.Core = require('./core');
```

Comment:
should we keep a `module.exports = require('./core');` to be backwards compatible?  @jcrugzz what do you think?